R/import_observations.R

```diff
@@ -34,6 +34,9 @@ import_observations <- function(observations, conn, max_dist = 336) {
   res <- dbSendQuery(conn, sql)
   dbClearResult(res)
 
+  # force observations into a data.frame to avoid problems with sf objects
```

Comment:
Why not derive x and y  automatically within the function if an sf object is used?

Something like ....
observations <- st_transform(observations, crs = 31370)
coords <- st_coordinates(observations)[,-3])
observations <- as.data.frame(observations)
observations$x <- coords[,1]
observations$y <- coords[,2]

Reply:
We could do that. Downside is that we need extra dependencies. You could give it a try. Here is an [example](https://github.com/ropensci/git2rdata/blob/master/R/write_vc.R) on how to select  a different function depending on the class of the input.